src/test/java/org/fundacionjala/coding/convert/DescendTest.java

```diff
@@ -0,0 +1,52 @@
+package org.fundacionjala.coding.convert;
+import org.junit.Assert;
+
+/**
+ * Test to Descend convertor.
+ */
+
+public class DescendTest {
+
+  /**
+   * test 1.
+   */
+  @org.junit.Test
+  public void sortedWordsDescend1() {
+
+    String text = "this kata is easy";
+    Descend descend = new Descend(text);
+    descend.sortTheInnerContent();
+
+    String textExpected = "tihs ktaa is esay ";
+    Assert.assertEquals(textExpected,descend.getResult());
+  }
+
+  /**
+   * test 2.
+   */
+  @org.junit.Test
+  public void sortedWordsDescend2() {
+
+    String text = "wait for me";
+    Descend descend = new Descend(text);
+    descend.sortTheInnerContent();
+
+    String textExpected = "wiat for me ";
+    System.out.println(descend.getResult());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)